htdocs/compta/bank/class/account.class.php

```diff
@@ -416,7 +416,7 @@ public function get_url($fk_bank = '', $url_id = '', $type = '')
 	 *  @param	int			$datev			Date value
 	 *  @return	int							Rowid of added entry, <0 if KO
 	 */
-    public function addline($date, $oper, $label, $amount, $num_chq, $categorie, User $user, $emetteur = '', $banque = '', $accountancycode = '', $datev = null)
+    public function addline($date, $oper, $label, $amount, $num_chq, $categorie, User $user, $emetteur = '', $banque = '', $accountancycode = '', $datev = null, $num_releve = '')
```

Comment:
If you add a parameter to a function, you must also add a line into the doc comment of function with @param (see other paramaters)

Reply:
Sorry, my first pull request. Will be more careful next time...